account_financial_report/report/aged_partner_balance_xlsx.py

```diff
@@ -96,61 +114,84 @@ def _get_report_columns(self, report):
                 "type": "amount",
                 "width": 14,
             },
-            8: {
-                "header": _("Current"),
-                "field": "current",
-                "field_footer_total": "current",
-                "field_footer_percent": "percent_current",
-                "field_final_balance": "current",
-                "type": "amount",
-                "width": 14,
-            },
-            9: {
-                "header": _("Age ≤ 30 d."),
-                "field": "30_days",
-                "field_footer_total": "30_days",
-                "field_footer_percent": "percent_30_days",
-                "field_final_balance": "30_days",
-                "type": "amount",
-                "width": 14,
-            },
-            10: {
-                "header": _("Age ≤ 60 d."),
-                "field": "60_days",
-                "field_footer_total": "60_days",
-                "field_footer_percent": "percent_60_days",
-                "field_final_balance": "60_days",
-                "type": "amount",
-                "width": 14,
-            },
-            11: {
-                "header": _("Age ≤ 90 d."),
-                "field": "90_days",
-                "field_footer_total": "90_days",
-                "field_footer_percent": "percent_90_days",
-                "field_final_balance": "90_days",
-                "type": "amount",
-                "width": 14,
-            },
-            12: {
-                "header": _("Age ≤ 120 d."),
-                "field": "120_days",
-                "field_footer_total": "120_days",
-                "field_footer_percent": "percent_120_days",
-                "field_final_balance": "120_days",
-                "type": "amount",
-                "width": 14,
-            },
-            13: {
-                "header": _("Older"),
-                "field": "older",
-                "field_footer_total": "older",
-                "field_footer_percent": "percent_older",
-                "field_final_balance": "older",
+        }
+        if not report.age_partner_config_id:
+            report_columns.update(
```

Comment:
Maybe it's better to use a hidden configuration (with active=False) with all these lines, and prevent its removal than hardcode all these lines. What do you think?